exporter/awscloudwatchlogsexporter/config_test.go

```diff
@@ -129,6 +129,70 @@ func TestRetentionValidateCorrect(t *testing.T) {
 
 }
 
+func TestInvalidRawLogEmfOnlyCombination(t *testing.T) {
+	defaultRetrySettings := exporterhelper.NewDefaultRetrySettings()
+	cfg := &Config{
+		RetrySettings:      defaultRetrySettings,
+		LogGroupName:       "test-1",
+		LogStreamName:      "testing",
+		Endpoint:           "",
+		LogRetention:       365,
+		AWSSessionSettings: awsutil.CreateDefaultSessionConfig(),
+		QueueSettings: QueueSettings{
+			QueueSize: exporterhelper.NewDefaultQueueSettings().QueueSize,
+		},
+		RawLog:  false,
+		EmfOnly: true,
+	}
+	assert.Error(t, component.ValidateConfig(cfg))
+}
+
+func TestValidRawLogEmfOnlyCombination(t *testing.T) {
+	defaultRetrySettings := exporterhelper.NewDefaultRetrySettings()
+	cfg := &Config{
+		RetrySettings:      defaultRetrySettings,
+		LogGroupName:       "test-1",
+		LogStreamName:      "testing",
+		Endpoint:           "",
+		LogRetention:       365,
+		AWSSessionSettings: awsutil.CreateDefaultSessionConfig(),
+		QueueSettings: QueueSettings{
+			QueueSize: exporterhelper.NewDefaultQueueSettings().QueueSize,
+		},
+		RawLog:  true,
+		EmfOnly: true,
+	}
```

Comment:
You could make a slice of the different test cases:
```
testCases = [][]bool{{true, true}, {false, false}, {true, false}}
```
then loop over the testCases. It would reduce some duplicate code.

Reply:
Agree with this, but instead of a []bool, would prefer
```go
testCases := []struct{
  RawLog bool
  EmfOnly bool
}{{true, true}, {false, false}, {true, false}}
```
so it's a bit clearer on first glance what those pairs are.

Reply:
This is a part of a much larger test case. I only added a few more cases. I don't think these flags relate to all the test cases only the ones I added.